src/components/StorageOverview/Utilization/Utilization.js

```diff
@@ -34,7 +34,7 @@ export const Utilization = ({ iopsUtilization, latencyUtilization, throughputUti
   return (
     <DashboardCard>
       <DashboardCardHeader>
-        <DashboardCardTitle>OCS Perfrormance</DashboardCardTitle>
+        <DashboardCardTitle>Perfrormance</DashboardCardTitle>
```

Comment:
These mockups have OCS Performance https://docs.google.com/presentation/d/15nDG3VuQRy6vJbMX4yq6ZvJ5G2Qawaz9glnLT8GEvX4/edit#slide=id.g4d3241551c_0_57 is there something newer ?

Reply:
@rawagner Yesterday we had a demo and we received this in the feedback. The feedback is collected in the doc[1].

https://docs.google.com/document/d/1fCLn9RM5yF5nvZuqqCDKdt-PzvjDohCkX1Jnh6yI560/edit